src/config/session-management.md

```diff
@@ -0,0 +1,38 @@
+# Session and Seat Management
+
+Session and seat management is not necessary for every setup, but it can be used
+to safely create temporary runtime directories, provide access to hardware
+devices and multi-seat capabilities, and control system shutdown.
+
+## D-Bus
+
+D-Bus is an IPC (inter-process communication) mechanism used by userspace
+software in Linux. There are two uses for it: as a system bus, and as a user
+bus.
+
+- To use it as a system bus, you should [enable](./services/index.md) the `dbus`
+   service. This might require a system reboot to work properly.
+- To use it as a user bus, you can start a given program (usually a window
+   manager or interactive shell) with
+   [dbus-run-session(1)](https://man.voidlinux.org/dbus-run-session.1). Most
```

Comment:
Just to check, are there actually any use-cases where `dbus-run-session(1)` *isn't* appropriate, such that users should use `dbus-launch(1)` instead? @abenson?

Reply:
Er, sorry, i meant @ahesford. But of course @abenson is welcome to chime in as well. :-)

Reply:
`dbus-launch` can be used without the `--exit-with-X11` argument in ways that `dbus-run-session` cannot, like launching a short-lived program but keeping the message bus alive. However, in the context of this document, the choice is really between `dbus-run-session` and `dbus-launch --exit-with-x11`. I don't believe there is any case where the latter would be preferred. For one, the `--exit-with-x11` option obviously requires X11. The `dbus-run-session` command is more general and, in my view, takes a more robust approach to lifetime management. (DBus is kept alive as long as the child, not when some second, single-purpose process loses its connection to an X server.)

Reply:
In that case, it makes sense to leave this item as is, right?

Reply:
@ericonr: On the basis of @ahesford's comment, yes, i think it can be left as-is.